samples/nextjs/src/pages/api/track/[action].ts

```diff
@@ -0,0 +1,17 @@
+import { createProxyMiddleware } from 'http-proxy-middleware';
+
+import jssConfig from 'temp/config';
```

Comment:
Maybe it worth to switch from configuration based settings to the env variables.
Potentially it might not work well  with vercel deployment.

Reply:
`temp/config.js` config supports environment variables.